README.md

```diff
@@ -66,10 +66,9 @@ def lambda_handler(event, context):
     corva.stream(stream_app, event)
 ```
 
-`Corva.stream` provides two optional parameters:
-
-- `filter_by_timestamp` - enable to clear [event](#event) from data with previously processed `timestamp`;
-- `filter_by_depth` - enable to clear [event](#event) from data with previously processed `measured_depth`.
+`Corva.stream` provides an optional parameter:
+- `filter_mode` - set to `timestamp` or `depth` to clear [event](#event) 
```

Comment:
Grammar correction:

> `filter-mode` - set to `timestamp` or `depth` to ignore [events](#event) that have been previously processed, based on the `timestamp` or `measured_depth` values.

Reply:
There was a mistake in readme, correct meaning is 

> `filter_mode` - set to `timestamp` or `depth` to clear [event](#event) datas with previously processed `timestamp` or `measured_depth`.

Is that good?

Reply:
added my description above, will change if needed

Reply:
`data` is already a plural word, so datas => data. Other than that, correct.